code/modules/mob/living/living.dm

```diff
@@ -777,6 +777,38 @@ Thanks.
 
 	var/mob/living/L = usr
 
+	//Escaping from within a subspace tunneler.
```

Comment:
At this moment in time? Probably not, other codebases have designed better systems though.